src/VecSim/algorithms/brute_force/brute_force.h

```diff
@@ -12,6 +12,7 @@ class BruteForceIndex : public VecSimIndex {
     size_t dim;
     VecSimType vecType;
     VecSimMetric metric;
+    friend class BF_BatchIterator;
```

Comment:
what for?

Reply:
For using the protected member `dim`. I thought that it is reasonable that the batch iterator class of the BF algorithm will be friend of the algorithm. but I can add a `getDim()` function instead